airbyte-webapp/src/views/Connection/CatalogDiffModal/components/DiffFieldTable.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../../../scss/variables";
+@use "./DiffSection.module.scss";
+
+.accordionSubHeader {
+  @extend .sectionSubHeader;
+  & th {
+    font-size: 10px;
+    width: 228px;
+    padding-left: variables.$spacing-lg;
+  }
+}
+
+.table {
+  width: 100%;
+  padding-left: variables.$spacing-xl;
+}
+
+tbody > tr:first-of-type > td:nth-of-type(2) {
+  border-radius: variables.$border-radius-sm variables.$border-radius-sm 0px 0px;
+}
+
+tbody > tr:last-of-type > td:nth-of-type(2) {
+  border-radius: 0px 0px variables.$border-radius-sm variables.$border-radius-sm;
+}
+
+tbody > tr:only-of-type > td:nth-of-type(2) {
+  border-radius: variables.$border-radius-sm;
+}
```

Comment:
Since there is no classname part of that and all selectors are matching on elements only, this should apply to all tabled on the whole page (once the modal has been loaded). I think we should make sure to nest this under a class that we actually apply in the component (which likely would also get rid of the eslint warnings you were seeing and ignored in the component).

Reply:
✅ nested